RFCs/FS-1063-support-letbang-andbang-for-applicative-functors.md

```diff
@@ -242,730 +208,30 @@ ce {
-
-## Run and Delay
-
-When a computation expression builder implements a `Run` or `Delay` method (or both), the desugared computation expression is wrapped in further calls corresponding to what is defined on the builder. Since applicative computations are required to follow the canonical form (exactly one `return`, etc.), an applicative computation expression will have precisely one `Run` method call (if it is defined) and one `Delay` method call (if it is defined).
-
-For example, if `Run` and `Delay` are both defined on the `ce` builder:
-
-```fsharp
-ce {
-    let! x = foo
-    and! y = bar
-    and! z = baz
-    return x + y + z
- }
-```
-
-becomes
-
-```fsharp
-builder.Run(
-    builder.Delay(fun () ->
-        ce.Apply(
-            ce.Apply(
-                ce.Apply(
-                    ce.Return(
-                        (fun x ->
-                            (fun y ->
-                                (fun z ->
-                                    x + y + z
-                                )
-                            )
-                        )),
-                    foo),
-                bar),
-            baz)
-    )
-)
-```
-
-These methods offer a mechanism to hook into before (`Delay`) and after (`Run`) the evaluation of the computation expression.
-
-### The proposed desugaring is purely syntactical
-
-The proposed change acts purely as a syntactic rewriting of a computation expression to calls to methods on a builder. As such, whilst the change is largely motivated by the theory of applicatives, the desugaring can be used to call methods of different types to those suggested above. This attribute is in line with the existing semantics of computation expression translation (note how [the MSDN docs](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/computation-expressions#creating-a-new-type-of-computation-expression) talk in terms of "_typical_ signatures").
 
 # Drawbacks
 [drawbacks]: #drawbacks
 
-The new applicative computation expressions are quite constrained, and as has been discussed, that is precisely what allows them to be so useful. However, these constraints are potentially somewhat unintuitive to the beginner. Computation expressions already involve one of the steeper learning curves of the F# language features, so the added complexity from this feature needs to be carefully weighed against the potential guarantees, expressiveness and performance gains that they can offer.
+TBD
 
 # Alternative Designs
 [alternative-designs]: #alternative-designs
 
+We chose not to support `use!` or `anduse!`. Why: TBD
```

Comment:
The `TBD` here will block on merging the feature, since we'll need a stated reason

Reply:
Yes I know